pkg/bpf/bpf.go

```diff
@@ -264,6 +279,82 @@ func recoverVersionMap(pinPath string) *ebpf.Map {
 	return versionMap
 }
 
+func (l *BpfLoader) setBpfProgOptions() {
+	nodeName := os.Getenv("NODE_NAME")
+	if nodeName == "" {
+		log.Error("skip kubelet probe failed: node name empty")
+		return
+	}
+
+	clientSet, err := kube.CreateKubeClient("")
+	if err != nil {
+		log.Errorf("get kubernetest client for getting node IP error: %v", err)
+		return
+	}
+
+	node, err := clientSet.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
+	if err != nil {
+		log.Errorf("failed to get node: %v", err)
+		return
+	}
+
+	// pass node ip and pod gateway to skip processing of kubelet access traffic.
+	nodeIP := getNodeIPAddress(node)
+	gateway := getNodePodSubGateway(node)
+
+	keyOfKmeshBpfConfig := uint32(0)
+	ValueOfKmeshBpfConfig := KmeshBpfConfig{
+		// Write this map only when the kmesh daemon starts, so set bpfloglevel to the default value.
+		BpfLogLevel: constants.BPF_LOG_INFO,
+		NodeIP:      nodeIP,
+		PodGateway:  gateway,
+	}
+
+	if l.kmeshConfig == nil {
+		log.Errorf("skip kubelet probe failed: %v", "kmeshConfigMap is nil")
+		return
+	}
+	if err := l.kmeshConfig.Update(&keyOfKmeshBpfConfig, &ValueOfKmeshBpfConfig, ebpf.UpdateAny); err != nil {
```

Comment:
Suggest: if if l.kmeshConfig != nil {

}, rather than printing the error log in L314